pkg/apis/meta/v1/deployment/deployments.go

```diff
@@ -0,0 +1,51 @@
+package deployment
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+//HaveDeploymentsInNamespace returns an error if all provided deployment are not installed in the given namespace
```

Comment:
the function name start with `have` give me a feel that it should return a bool?

Reply:
Done